transcom/mymove
# B19765 int 20240507 fe change text auth receipt total

## Agility ticket
[B19765](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19765)

## Summary

In "Review Documents" page in the "Summary", “Authorized Receipt Total” was changed to “Accepted Receipt Totals”.

### How to test

* Customer:  create a PPM move.
* SC: login and approve it.
* Customer: Upload PPM documents.  Upload expense receipts.
* SC: Review Documents.  Get to the summary screen.  Will see “Accepted Receipt Totals”.

## Screenshots
![image](https://github.com/transcom/mymove/assets/146897935/e79a2d09-59d3-45c0-8160-ef4b5ac2b7f1)
